.github/workflows/main.yml

```diff
@@ -26,13 +26,23 @@ jobs:
       - name: Install tox
         run: |
           python -m pip install -U tox
-      - name: Build example docs
-        run: tox run -e py
-      - name: Compress Artifacts
+      # For some reason, the Jupyter Extension fails to build properly if one theme has already
+      # been built with a custom BUILD_DIR. So, we eagerly upload the artifacts for each theme
+      # build, and then delete the BUILD_DIR before proceeding to the next theme.
+      - name: Create artifacts/ folder
+        run: mkdir artifacts
+      - name: Build Legacy theme
         run: |
-          mkdir artifacts
-          tar -zcvf html_docs.tar.gz example_docs/docs/_build/html
-          mv html_docs.tar.gz artifacts/.
+          THEME=qiskit_sphinx_theme BUILD_DIR=example_docs/_build_legacy/html tox run -e py
+          tar -zcvf legacy_html_docs.tar.gz example_docs/_build_legacy/html
+          mv legacy_html_docs.tar.gz artifacts/.
+          rm -rf example_docs/_build_legacy
+      - name: Build Furo theme
+        run: |
+          THEME=_qiskit_furo BUILD_DIR=example_docs/_build_furo/html tox run -e py
```

Comment:
I don't fully understand the first sentence.

But this theme name is temporary. Once the theme is stable, we will rename it to `qiskit` and `qiskit_ecosystem`, without `furo` in the name. This temporary theme name `_qiskit_furo` is only to allow us to safely experiment while developing it.

It doesn't really matter what we call `_qiskit_furo` since it is temporary.